plugins/woocommerce-admin/client/launch-your-store/tour/use-site-visibility-tour.tsx

```diff
@@ -31,15 +31,18 @@ export const useSiteVisibilityTour = () => {
 		};
 	} );
 
-	const onClose = () => {
+	const onClose = async () => {
 		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 		// @ts-ignore
-		dispatch( 'core' ).saveUser( {
+		await dispatch( 'core' ).saveUser( {
 			id: window?.wcSettings?.currentUserId,
 			meta: {
 				woocommerce_launch_your_store_tour_hidden: 'yes',
```

Comment:
There's a `const` for this key above, may as well use it here too.

Reply:
Sounds good! Updated in e8047cfe2d294df02065eab9e880379de3785a0e